cprover_bindings/src/goto_program/typ.rs

```diff
@@ -727,6 +784,7 @@ impl Type {
     /// Since they have different padding.
     /// https://github.com/diffblue/cbmc/blob/develop/src/solvers/lowering/byte_operators.cpp#L1093..L1136
     pub fn is_structurally_equivalent_to(&self, other: &Type, st: &SymbolTable) -> bool {
+        let other = other.unwrap_typedef();
```

Comment:
This is actually redundant, since all the other functions are unwrapping the element. I just figured it was worth only unwrapping it once. But it doesn't make sense that it's not symmetric, eh? Let me fix that.